spec/features/events_spec.rb

```diff
@@ -146,17 +146,83 @@
 
   scenario "logged in as Organizer I can push the accept all button to accept all applicants" do
     login(:organizer)
-    @event.max_participants = 3
-    2.times do |i|
-      @pupil = FactoryGirl.create(:profile)
-      @application_letter = FactoryGirl.create(:application_letter, event: @event, user: @pupil.user)
-    end
+    @event = FactoryGirl.create :event, :with_diverse_open_applications, participants_are_unlimited: true
     visit event_path(@event)
     click_link I18n.t "events.applicants_overview.accept_all"
     application_letters = ApplicationLetter.where(event: @event.id)
     expect(application_letters.all? { |application_letter| application_letter.status == 'accepted' }).to eq(true)
   end
 
+  scenario "logged in as Organizer and viewing the participants page all checkboxes are checked when pressing the \"check all\" button", js: true do
+    login(:organizer)
+    @user = FactoryGirl.create(:user)
+    @profile = FactoryGirl.create(:profile, user: @user, birth_date: 15.years.ago)
```

Comment:
Why would you make the user underaged? Wouldn't it be easier and more understandable if the user was an adult, thus no agreement letter is needed?